libs/klv/src/main/java/org/codice/alliance/libs/klv/GeoBoxHandler.java

```diff
@@ -156,6 +151,18 @@ public String getLongitude4() {
         return asAttribute(polygonsWkts);
     }
 
+    /**
+     * Trim the arrays of lat and lon values to the same length.
+     */
+    @Override
+    public void trim() {
+        Trimmable.trimList(map.values(), getMinimumListSize());
```

Comment:
The class is modifying it's own internal data, which is really no different if it were to add values to it's internal data. These KlvHandlers build up data is the raw klv data is feed to them. I'd be happy to meet with you and talk about the code.